www/web-proxy-useracl/src/opnsense/mvc/app/models/OPNsense/ProxyUserACL/ProxyUserACL.xml

```diff
@@ -1,25 +1,239 @@
+                            <F>Friday</F>
+                            <A>Saturday</A>
+                        </OptionValues>
+                    </Days>
+                    <Start type="TextField">
+                        <default>0</default>
+                        <Required>Y</Required>
+                    </Start>
+                    <End type="TextField">
+                        <default>0</default>
+                        <Required>Y</Required>
+                    </End>
+                    <id type="AutoNumberField">
+                        <Required>Y</Required>
+                    </id>
+                    <Description type="TextField">
+                        <Required>Y</Required>
+                    </Description>
+                </Time>
+            </Times>
+            <Mimes>
+                <Mime type="ArrayField">
+                    <Names type="CSVListField">
+                        <Required>Y</Required>
+                    </Names>
+                    <id type="AutoNumberField">
+                        <Required>Y</Required>
+                    </id>
+                    <Description type="TextField">
+                        <Required>Y</Required>
+                    </Description>
+                </Mime>
+            </Mimes>
+            <Srcs>
+                <Src type="ArrayField">
+                    <Names type="CSVListField">
+                        <mask>/^([\/0-9a-fA-F.:,])*/u</mask>
+                        <Required>Y</Required>
+                    </Names>
+                    <id type="AutoNumberField">
+                        <Required>Y</Required>
+                    </id>
+                    <Description type="TextField">
+                        <Required>Y</Required>
+                    </Description>
+                </Src>
+            </Srcs>
+            <Dsts>
+                <Dst type="ArrayField">
+                    <Names type="CSVListField">
+                        <mask>/^([\/0-9a-fA-F.:,])*/u</mask>
+                        <Required>Y</Required>
+                    </Names>
+                    <id type="AutoNumberField">
+                        <Required>Y</Required>
+                    </id>
+                    <Description type="TextField">
+                        <Required>Y</Required>
+                    </Description>
+                </Dst>
+            </Dsts>
+            <Arps>
+                <Arp type="ArrayField">
+                    <Names type="CSVListField">
+                        <mask>/^([\/0-9a-fA-F:,\-])*/u</mask>
+                        <Required>Y</Required>
+                    </Names>
+                    <id type="AutoNumberField">
+                        <Required>Y</Required>
+                    </id>
+                    <Description type="TextField">
+                        <Required>Y</Required>
+                    </Description>
+                </Arp>
+            </Arps>
```

Comment:
if possible, it would be better to rename that to MAC addresses since it is not ARP - the name would also conflict with NDP (IPv6).